packages/nitro-protocol/deployment/deploy.ts

```diff
@@ -56,6 +57,11 @@ export async function deploy(): Promise<Record<string, string>> {
     {},
     TEST_NITRO_ADJUDICATOR_ADDRESS
   );
+  const TEST_ETH_ASSET_HOLDER2_ADDRESS = await deployer.deploy(
```

Comment:
Is there a risk of `deploy.js` being used in "production"? (to deploy to mainnet, lets say?)

The `TestEthAssetHolder` contracts now have `setHoldings`, which mean we only want to deploy it to a test environment. The reason for asking then, is to determine: how can we ensure that `TestEthAssetHolder` is only used in a test environment?

Reply:
I'm going to address this in a separate PR.